app/_locales/es/messages.json

```diff
@@ -238,24 +424,56 @@
   "customRPC": {
     "message": "RPC personalizado"
   },
+  "customSpendLimit": {
+    "message": "Límite de Gasto Personalizado"
+  },
   "customToken": {
     "message": "Token Personalizado"
   },
+  "dataBackupFoundInfo": {
+    "message": "Se hizo una copia de seguridad de algunos de los datos de su cuenta durante una instalación anterior de MetaMask. Esto podría incluir su configuración, contactos y tokens. ¿Le gustaría restaurar estos datos ahora?"
+  },
   "decimal": {
     "message": "Decimales de precisión"
   },
   "decimalsMustZerotoTen": {
     "message": "Los decimales deben ser al menos 0 y no más de 36"
   },
+  "decrypt": {
+    "message": "Descifrar"
+  },
+  "decryptCopy": {
+    "message": "Copiar mensaje cifrado"
```

Comment:
Shouldn't be instead "Copiar mensaje decifrado"?

Reply:
I'm not sure about this, the [english version](https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/en/messages.json#L449) is "Copy encrypted message". So I guess "Copiar mensaje cifrado" or "Copiar mensaje encriptado" should be ok.

Reply:
got confused by the id then. in that case, is ok.